core/src/avm2/globals/flash/sensors/Accelerometer.as

```diff
@@ -0,0 +1,10 @@
+package flash.sensors {
+    import __ruffle__.stub_getter;
+
+    public class Accelerometer {
+        public static function get isSupported():Boolean {
+            stub_getter("flash.sensors.Accelerometer", "isSupported");
```

Comment:
I don't think we need to mark this as a stub really, it's correct - we don't support it :D